back/node_watcher/src/tasks/types.ts

```diff
@@ -123,13 +124,28 @@ export enum ProposalStatus {
   TABLED = 'Tabled',
 }
 
-export enum ReferendumStatus {
-  CANCELLED = ' Cancelled',
-  EXECUTED = 'Executed',
-  NOTPASSED = 'NotPassed',
-  PASSED = 'Passed',
-  STARTED = 'Started',
-}
+export const referendumStatus = {
+  CANCELLED: 'Cancelled',
+  EXECUTED: 'Executed',
+  NOTPASSED: 'NotPassed',
+  PASSED: 'Passed',
+  STARTED: 'Started',
+};
+
+// export type ReferendumStatus =
+//   | 'Cancelled'
+//   | 'Executed'
+//   | 'NotPassed'
+//   | 'Passed'
+//   | 'Started';
+
```

Comment:
I used to have an enum, but it doesn't allow to do the following because of type missmatch string != ReferendumStatus :
```js
Object.values(referendumStatus)
.filter(status => status !== referendumStatus.STARTED)
.includes(method)
```